receiver/opencensus/opencensus_test.go

```diff
@@ -174,6 +177,101 @@ func TestGrpcGatewayCors_endToEnd(t *testing.T) {
 	verifyCorsResp(t, url, "disallowed-origin.com", 200, false)
 }
 
+// As per Issue https://github.com/census-instrumentation/opencensus-service/issues/366
+// the agent's mux should be able to accept all Proto affiliated content-types and not
+// redirect them to the web-grpc-gateway endpoint.
+func TestAcceptAllGRPCProtoAffiliatedContentTypes(t *testing.T) {
+	t.Skip("Currently a flaky test as we need a way to flush all written traces")
```

Comment:
If you add a short wait for the server to start like the test above for the grpc-gateway, will that work?

The way that the test above at least attempts to flush all written traces is by using the `octrace.WithSpanBufferCount(1)`. Does that not work here?

Reply:
@draffensperger, yes I did that if you check in the code below.